docs/adrs/0008-appliance-calculator-location.md

```diff
@@ -0,0 +1,83 @@
+# 8. Appliance Calculator Code Location
+
+Date: 2024-10-15
+
+## Status
+
+Draft, Proposed
+
+## Context
+
+The energy team's appliance calculator is currently served be EpiServer at https://www.citizensadvice.org.uk/consumer/energy/energy-supply/save-energy-at-home/check-how-much-your-electrical-appliances-cost-to-use/.  
+
+The Digital Energy Team is rebuilding this app as a progressively enhanced Rails app and we need to decide where the code for that app should live.  
+
+The energy team currently have three code repos for their apps in production:
+- [citizensadvice/energy-prototypes](https://www.github.com/citizensadvice/energy-prototypes) (current appliance calculator, will be archived on completion of this work)
+- [citizensadvice/smart-meter-tool](https://github.com/citizensadvice/smart-meter-tool) (has specific contractual deployment and hosting isolation requirements)
+- [citizensadvice/energy-comparison-table](https://github.com/citizensadvice/energy-comparison-table) (single rails app handling requests for the customer service rating comparison table)
+
+The appliance calculator itself will be rendered into an iframe on an advice collection page by the public website, which involves creating a [new content type](https://app.contentful.com/spaces/vms0u05139aw/environments/qa/content_types/interactiveTool/fields) to represent the calculator, making content-api understand that new content type and creating a new renderer in the public website codebase.  Rough implementation spikes have proved that this will work:
```

Comment:
Have we confirmed with AbilityNet that it's okay to use iframes?

Reply:
I would imagine it would generally be fine - conveniently, there is a drop in session next Monday (28th Oct) so could be worth bringing up there.

Reply:
Thanks! I'll bring it up at the session.  I think they aren't inherently inaccessible - from what I remember it's important to let people scroll in them and for them to have a good title.